src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -39,12 +39,13 @@ namespace app {
 namespace Clusters {
 namespace MicrowaveOvenControl {
 
-Instance::Instance(Delegate * aDelegate, EndpointId aEndpointId, ClusterId aClusterId,
+Instance::Instance(Delegate * aDelegate, EndpointId aEndpointId, ClusterId aClusterId, uint32_t aFeature,
```

Comment:
Is there a reason aFeature is not of type `MicrowaveOvenControl::Feature` or `BitFlags<MicrowaveOvenControl::Feature>` or some such?